content/concept-money.md

```diff
@@ -0,0 +1,43 @@
+# Money in the XRP Ledger
+
+The XRP Ledger is an advanced blockchain-like system that was designed to let people transact in multiple currencies. Users of the XRP Ledger can seamlessly track, trade, and settle multiple currencies in an exchange that's as decentralized as the network itself. Users can issue their own currency-like digital assets, which may or may not represent obligations owed outside the XRP Ledger. Tying it all together is XRP, the XRP Ledger's native cryptocurrency, which serves anti-spam purposes and acts as a medium of exchange.
```

Comment:
Consider flipping `anti-spam purposes` and `acts as a medium of exchange`. The current order might give the impression that anti-spam is the more important use case.